src/core/tools/attemptCompletionTool.ts

```diff
@@ -76,13 +76,13 @@ export async function attemptCompletionTool(
 				}
 
 				// Complete command message.
-				const executionId = Date.now().toString()
-				const didApprove = await askApproval("command", command, { id: executionId })
+				const didApprove = await askApproval("command", command)
 
 				if (!didApprove) {
 					return
 				}
 
+				const executionId = cline.lastMessageTs?.toString() ?? Date.now().toString()
```

Comment:
This is a better value for `executionId` since it will get passed as `message.ts` for the `command` message. I was noticing issues with existing method for passing an `executionId`.